plugin-code/src/main/java/com/trivago/cluecumber/json/pojo/Embedding.java

```diff
@@ -52,13 +56,49 @@ public void setFilename(final String filename) {
     public boolean isImage() {
         return
                 mimeType.equalsIgnoreCase("image/png") ||
```

Comment:
It would make sense to have the mime types as an enum since they are reused in some places throughout your code.

Reply:
Added MimeType.java as enum for MimeTypes